mcs/class/System.Drawing/Makefile

```diff
@@ -5,6 +5,7 @@ SUBDIRS =
 LIBRARY = System.Drawing.dll
 
 LIB_REFS = System
+KEYFILE = ../msfinal.pub
```

Comment:
you can get rid of the ifdef MOBILE_LIKE on line 17 below now.

Though this does change the logic a bit since we only set the keyfile on !MOBILE in AssemblyInfo.cs before. I'm on my phone right now so can't easily check whether it has an impact, will do later at home :)